src/Sulu/Bundle/AdminBundle/Resources/js/containers/SingleSelection/SingleSelection.js

```diff
@@ -146,16 +150,28 @@ class SingleSelection extends React.Component<Props> {
                     value={item}
                 >
                     {item &&
-                        <div>
-                            {displayProperties.map((displayProperty) => (
-                                <span
-                                    className={singleSelectionStyles.itemColumn}
-                                    key={displayProperty}
-                                    style={{width: 100 / columns + '%'}}
-                                >
-                                    {item[displayProperty]}
-                                </span>
-                            ))}
+                        <div className={singleSelectionStyles.itemContainer}>
+                            {(publishedState !== undefined || published !== undefined)
+                            && !(publishedState && published) &&
```

Comment:
Here I see better than above, that the indentation is not correct... I would have written it this way:

```javascript
{(publishedState !== undefined || published !== undefined) &&
    !(publishedState && published) &&
    <div></div>
}
```